src/DynamoCore/Graph/Nodes/NodeInputData.cs

```diff
@@ -104,6 +138,8 @@ public class NodeInputData
             { typeof(Int64),NodeInputTypes.numberInput},
             {typeof(float),NodeInputTypes.numberInput},
         };
+
+        [Obsolete("To be removed in Dynamo 3.x")]
```

Comment:
Why is this being obsoleted?

Reply:
It's a public method, but it looks like an implementation detail...

Reply:
If it's going to be replaced with a method having a different name or made internal that's fine. After seeing the obsolete attribute I thought it was no longer needed but was confused since it's still being used in a number of places.